### ISSUE TITLE
Drag and drop between browser tabs

### ISSUE THREAD

**Issue by temka1234** [caplin/FlexLayout#203]
Hi! Is there any plan to make dnd between browser tabs?

**Comment by edemaine** [comment]
We recently added functionality to the `master` branch to drag **links** from another tab into a FlexLayout.  (Check out the `onExternalDrag` configuration option and try out [the demo](https://rawgit.com/caplin/FlexLayout/demos/demos/v0.5/demo/index.html).)  But I'm guessing you mean dragging FlexLayout tabs between different FlexLayouts possibly in different browser tabs?  That's a neat idea too...

**Comment by temka1234** [comment]
Yes, want to make single dashboard on multiple tabs, connected through shared worker.  
I think its possible with native drag and drop API.
